subsys/bluetooth/mesh/Kconfig

```diff
@@ -814,10 +814,17 @@ config BT_MESH_SEQ_STORE_RATE
 	  off with a value that's guaranteed to be larger than the last
 	  one used before power off.
 
+config BT_MESH_RPL_STORE
+	bool "Enable storing RPL entries"
+	default y
```

Comment:
Should have a `depends on BT_SETTINGS`

Reply:
This is already located within
``` 
if BT_SETTINGS
....
endif # BT_SETTINGS
```
section